frame/contracts/src/wasm/runtime.rs

```diff
@@ -571,6 +586,30 @@ define_env!(Env, <E: Ext>,
 		}
 	},
 
+	// Remove the calling account and transfer remaining balance.
+	//
+	// This function never returns. Either the termination was successful and the
+	// execution of the destroyed contract is halted. Or it failed during the termination
+	// which is considered fatal and results in a trap + rollback.
+	//
+	// - beneficiary_ptr: a pointer to the address of the beneficiary account where all
+	//   where all remaining funds of the caller are transfered.
+	//   Should be decodable as an `T::AccountId`. Traps otherwise.
+	// - beneficiary_len: length of the address buffer.
+	ext_terminate(
+		ctx,
+		beneficiary_ptr: u32,
+		beneficiary_len: u32
```

Comment:
```suggestion
		beneficiary_len: u32,
```

Reply:
The macro where this code is in does not support trailing comma.

Reply:
in hindsight, I think it is the time to actually add support for that feature. There was uncountable number of times when people suggested that, then optionally the author tried to add that or maybe applied the suggestion, and then had to report back and possibly revert the suggestion.